### DISCUSSION TITLE
docker

### DISCUSSION THREAD

**Discussion by 72econoline** [tphakala/birdnet-go#255] [Q&A]
Thanks for this program, I really do like it!

I am new to docker, never really looked into it before now.  I wanted birdnet-go docker to run on boot-up, without logging into my PiZero.  After some searching a bit I found and ran these commands:
sudo systemctl enable docker.service
sudo systemctl enable containerd.service
sudo docker run --restart=always -d ghcr.io/tphakala/birdnet-go:latest

It seems to be running OK, starts on boot.  My question is, it seems there is more than one container running?
pi@birdpi:~ $ docker stats --format "table {{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}"
NAME                    CPU %     MEM USAGE / LIMIT
gifted_banach           --        -- / --
compassionate_hellman   --        -- / --

but running with --no-stream option I just see one:
pi@birdpi:~ $ docker stats --format "table {{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}" --no-stream
NAME            CPU %     MEM USAGE / LIMIT
gifted_banach   120.29%   0B / 0B

pi@birdpi:~ $ docker ps
CONTAINER ID   IMAGE                                COMMAND                  CREATED        STATUS          PORTS                                       NAMES
a6e116c07346   ghcr.io/tphakala/birdnet-go:latest   "/usr/bin/birdnet-go…"   42 hours ago   Up 21 seconds   8080/tcp                                    compassionate_hellman
c00e9c9f2a75   ghcr.io/tphakala/birdnet-go:latest   "/usr/bin/birdnet-go…"   42 hours ago   Up 44 minutes   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   gifted_banach

Is this normal?  Hows it look?

Thanks

**Comment by 72econoline** [comment]
I ended up stopping a6e116c07346 
I recommend other new docker users read this:
https://www.freecodecamp.org/news/how-docker-containers-work/

Thanks

**Reply by Subr1ata** [reply]
Are you still looking for a help ?

**Reply by 72econoline** [reply]
Thanks!  I got it going OK.  I just had to learn about docker and how to use it.
